CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.37.0
+* Bug - fixed a bug where Firelens container could not use config file from S3 bucket in us-east-1 [#2356](https://github.com/aws/amazon-ecs-agent/pull/2356)
+* Enhancement - additional parameters allowed when specifying secretsmanager secret [#2358](https://github.com/aws/amazon-ecs-agent/pull/2358)
```

Comment:
nit: seems like the convention is to put Feature at top, then Enhancement, then Bug

Reply:
also the secret part should be Feature, otherwise you shouldn't bump the major version

Reply:
updated